src/containers/LoginForm/styles.module.scss

```diff
@@ -0,0 +1,25 @@
+@import "@scss/mixins";
+
+.container {
+    @include flex-col-center;
+    height: 100vh;
+}
+.form {
+    border-radius: 5px;
+    background-color: #fff;
+    padding: 20px;
+    width: 100%;
+    position: relative;
+}
+.row {
+    margin-bottom: 20px;
+
+    &.heading {
+        @include flex-center-between;
+    }
+
+    &:last-child {
+        display: grid;
+        place-items: center;
```

Comment:
another `grid-center` mixin.

Reply:
I have changed it, please review my change